font-patcher

```diff
@@ -904,6 +904,21 @@ class font_patcher:
             self.sourceFont.hhea_ascent = self.sourceFont.os2_winascent
             self.sourceFont.hhea_descent = -self.sourceFont.os2_windescent
 
+    def add_glyphrefs_to_essential(self, unicode):
+        self.essential.add(unicode)
+        # According to fontforge spec, altuni is either None or a tuple of tuples
+        if self.sourceFont[unicode].altuni is not None:
+            for r in self.sourceFont[unicode].altuni:
```

Comment:
Right, it is indeed arbitrary -- I "chose" `r` because it was already used in `get_essential_references()`, so I just used the same variable. `r` here simply refers to one of the altuni bindings currently being iterated. What would you suggest I call the variable?